app/career/views/weekly_business.py

```diff
@@ -0,0 +1,101 @@
+from django.db.models import Q
+from django.utils.translation import gettext as _
+from rest_framework import status, viewsets
+from rest_framework.response import Response
+
+from sentry_sdk import capture_exception
+
+from app.career.models import WeeklyBusiness
+from app.career.serializers import WeeklyBusinessSerializer
+from app.common.drive_handler import upload_and_replace_image_with_cloud_link
+from app.common.enums import AppModel
+from app.common.pagination import BasePagination
+from app.common.permissions import IsNoK
+from app.util import today, week_nr
+
+
+class WeeklyBusinessViewSet(viewsets.ModelViewSet):
+
+    queryset = WeeklyBusiness.objects.none()
+    serializer_class = WeeklyBusinessSerializer
+    permission_classes = [IsNoK]
+    pagination_class = BasePagination
+
+    def get_queryset(self):
+        """
+        Return all non-expired weekly_business by default.
+        Filter expired events based on url query parameter.
+        """
+        if "expired" in self.request.query_params:
+            return WeeklyBusiness.objects.all().order_by("year", "week")
+        if "this_week" in self.request.query_params:
+            return WeeklyBusiness.objects.filter(
+                year=today().year, week=week_nr(today())
+            )
+        in_future_this_year_filter = Q(year=today().year) & Q(
+            week__gte=week_nr(today())
+        )
+        next_year_filter = Q(year__gt=today().year)
+        return WeeklyBusiness.objects.filter(
+            (in_future_this_year_filter) | next_year_filter
+        ).order_by("year", "week")
```

Comment:
Bare nysgjerrig: Hva skjer her? Får man bedriftene for dette og neste år?

Reply:
Dette er for å garantere at jeg henter ut ting for fremtiden. Så hente ut alle uker i fremtiden dette året og alle de neste årene